bats/sql-add.bats

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+
+    dolt sql <<SQL
+CREATE TABLE test (
+    pk int primary key
+);
+CREATE TABLE test2 (
+    pk int primary key
+);
+INSERT INTO test VALUES (0),(1),(2);
+SQL
+    dolt sql <<SQL
```

Comment:
This second block doesn't seem to be doing anything useful. Just create the table with the data you want